workspace/welcome-to-xod/014-internet/patch.xodp

```diff
@@ -0,0 +1,128 @@
+{
+  "comments": [
+    {
+      "content": "# Internet\n\nThere is a large class of tasks, called Internet-Of-Things when the hardware needs an Internet connection to send or receive data from some server. For example, log the temperature and moisture for smart greenhouses, getting the current time, weather, work with XOD Cloud or other APIs, and so on.\n\nIn this chapter, we get the current date and time from the XOD Cloud API.\n\nFor this, there is the `xod-cloud/basics/now` node, which sends a request to the XOD Cloud, parses, and outputs the response as `datetime` type value.\n\nThis node has an input pin `INET`. This pin has to be connected with an internet provider. The Internet can be provided\n- with a microcontroller itself\n  `xod-dev/esp8266-mcu/connect`,\n- with an external module\n  `xod-dev/esp8266/connect`,\n- with a shield\n  `xod-dev/w5500/ethernt-shield`,\n- *from the PC*\n  `xod/debug/tethering-inet`.\n  It works either in simulation or debug mode.",
+      "id": "HyGHq2IhU",
+      "position": {
+        "units": "slots",
+        "x": 1,
+        "y": 0
+      },
+      "size": {
+        "height": 6,
+        "units": "slots",
+        "width": 7
+      }
+    },
+    {
+      "content": "Note, we got a `select` node here to get the reasonable value until we get a result from the cloud. Otherwise, we would get a default datetime value, which is equal to `1970-01-01 00:00:00`, and it might be confusing.\n\n## Excercise\nProvide the Internet from your PC.\n1. Place `xod/debug/tethering-inet` node.\n2. Link the `tethering-inet` node with the `INET` pin of the `now` node.\n3. Simulate the patch and take a look at the obtained datetime.",
```

Comment:
The tutorial conventionally has all the starting patches functional. Adding the `tethering-inet` should not be an exercise. If an exercise is desired, I’d better offer to add a `tweak-pulse` to update the time in runtime.